code/obj/item/mine.dm

```diff
@@ -201,7 +201,7 @@
 		if (length(mobs))
 			for (var/mob/living/L in mobs)
 				if (istype(L))
-					L.changeStatus("radiation", 60 SECONDS)
+					L.take_radiation_dose(5 SIEVERTS)
```

Comment:
😰 😰 😰 😰 😰

Reply:
this went from "take ~70 damage over 60 seconds" to "you die before 60 seconds"

Reply:
I'm nerfing it from crazy, but I'm still going to make it a potentially lethal dose because dammit radiation landmines should be harmful!